zipline/gens/brokers/ib_broker.py

```diff
@@ -192,7 +192,10 @@ def subscribe_to_market_data(self,
         self.ticker_id_to_symbol[ticker_id] = symbol
 
         tick_list = "233"  # RTVolume, return tick_type == 48
-        self.reqMktData(ticker_id, contract, tick_list, False)
+        if contract.m_secType == 'IND':
+            self.reqRealTimeBars( ticker_id, contract, 60, 'TRADES', True)
```

Comment:
Since IB has pace limitation on reqRealtimeBars it is possible that this request runs to an error. Safest would be to add a sleep(11) after reqRealtimeBars() call.